packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+
+| Property | Type    | Description                                                                              |
+|----------|---------|------------------------------------------------------------------------------------------|
+| id       | String  | Unique identifier for this Policy (read-only).                                           |
+| name     | String  | Human-readable name for the Policy, configurable during creation or updating.            |
+| type     | String  | Type of the policy. For IdP Routing Policy objects, this needs to be `Okta:UnknownUser`. |
+| status   | String  | `ACTIVE`  or  `INACTIVE`.                                                                |
+| default  | Boolean | `true` for the first instance of this policy, which gets created by default.             |
+
+### Unknown User Policy Object Example
+
+```json
+{
+    "id": "rst10y3icArzJtWvf0g4",
+    "name": "Default Policy",
+    "type": "Okta:UnknownUser",
+    "status": "ACTIVE",
+    "default": true,
+    "_links": {
+        "self": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10y3icArzJtWvf0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "rules": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10y3icArzJtWvf0g4/rules",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+
+## Unknown User Rule Object
+
+| Property    | Type                                                                         | Description                                                                                        |
+|-------------|------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| name        | String                                                                       | Human-readable name for the Rule, configurable during creation or updating.                        |
+| id          | String                                                                       | Unique identifier for this Rule (read-only)                                                        |
+| type        | String                                                                       | Type of the Rule. For Identifier Match Rule Objects, this needs to be `Okta:UnknownUser`.          |
+| priority    | Integer                                                                      | Used to determine which Rules take precedence.                                                     |
+| conditions  | Array                                                                        | No conditions are supported for this Rule type, so this must be an empty array.                    |
```

Comment:
it has same `conditions` as identify matching rule. @larsjohansen-okta @iancorcoran-okta correct me wrong.

Reply:
You're right - in the backend code this is odd since we're missing a condition validator for this policy so I assumed that none were allowed but turns out that wasn't the case. I will file a JIRA to fix that, and give examples in the API to reflect what is supported in the UI. @motikagan-okta can you change the text to match what we have for Identifier First policy rule conditions, and also include this in the example below:

```
    "conditions": [
        {
            "key": "Okta:Identifier",
            "op": "STRING_ENDS_WITH",
            "value": "idx.com"
        },
        {
            "key": "Okta:AppInstance",
            "op": "IN_LIST",
            "value": [
            	"{{appInstanceId}}"
        	]
        }
    ],
```?

Reply:
Yeah. It looks like the backend validation for this was missed when it was implemented, but we can doc that those conditions are supported

Reply:
Added the information and example for Identifier and App Instance conditions to the Identifier Match Rule.